File: sub-web-model/src/main/java/org/github/base/mapper/ISysUserMapper.java
package org.github.base.mapper;

import org.github.base.entity.SysUserEntity;
import org.github.base.IMapper;
import org.github.mybatis.MyBatisMapper;

/**
 * <p>
 * 系统用户 Mapper 接口
 * </p>
 *
 * @author JYD_XL
 * @since 2021-05-07
 */
@MyBatisMapper
public interface ISysUserMapper extends IMapper<SysUserEntity> {}

File: sub-mysql-mydb-service/src/main/java/org/github/mysql/mydb/base/mapper/ISysUserMapper.kt
@file:Suppress("RedundantSemicolon")

package org.github.mysql.mydb.base.mapper;

import org.github.mysql.mydb.base.entity.SysUserEntity;
import org.github.base.IMapper;
import org.github.mybatis.MyBatisMapper;

/**
 * 用户表 Mapper 接口
 *
 * @author JYD_XL
 */
@MyBatisMapper
interface ISysUserMapper : IMapper<SysUserEntity>
